stories/Molecules/Imagecaption/Imagecaption.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './imagecaption.scss';
+import { Imagecredit } from '../../Atom/Images/Imagecredit/Imagecredit';
+
+export const Imagecaption = ({ name, label, paragraph, ...props }) => {
+    return (
+        <div className="image-caption">
+            <figcaption className="image__caption__para">
+                { paragraph }
+            </figcaption>
+            <Imagecredit label={ label } name={ name }/>
+        </div>
+    );
+};
```

Comment:
@Shubham-Sharma-77 figcaption will wrap the complete info that it including imagecredit